examples/load_balancer/README.md

```diff
@@ -17,37 +17,37 @@ An example usage for LB server port at 10.0.0.37 with clients matching 11.0.0.0/
 ```sudo ip route add 11.0.0.0/24 via 10.0.0.37 dev p2p1```  
 
 **This NF should be run with the ARP NF**    
-The Load Balancer NF needs to respond to client and server ARP requests. As `onvm_mgr` currently doesn't resolve arp requests, an ARP NF    with the LB NF as destination is used.
+The Load Balancer NF needs to respond to client and server ARP requests. As `onvm_mgr` currently doesn't resolve arp requests, an ARP NF with the LB NF as destination is used.
 
-An example usage of the ARP&LB NF, with the Load Balancer using dpdk0 - 10.0.0.37 and  dpdk1 - 11.0.0.37 for client, server ports respecively. 
+An example usage of the ARP&LB NF, with the Load Balancer using interface-1 (IP 10.0.0.37, port0) and interface-2 (IP 11.0.0.37, port1) for client and server ports, respecively. 
 ```
 ARP NF
 ./go.sh 1 -d 2 -s 10.0.0.37,11.0.0.37
 
 LB NF
-./go.sh 2 -c dpdk0 -s dpdk1 -f server.conf 
+./go.sh 2 -c 10.0.0.37/0 -s 11.0.0.37/1 -f server.conf 
 ```
 
 
 Compilation and Execution
 --
 ```
 make
-./go.sh SERVICE_ID -c CLIENT_IFACE -s SERVER_IFACE -f SERVER_CONFIG [PRINT_DELAY]
+./go.sh SERVICE_ID -c CLIENT_INFO -s SERVER_INFO -f SERVER_CONFIG [PRINT_DELAY]
```

Comment:
Would it make more sense to just call this CLIENT/SERVER_IP?

Reply:
Given that the port ID must also be defined alongside the IP, I didn't want to make it appear that only the IP is required. I figured `CLIENT/SERVER_INFO` would then require users to check the `App Specific Arguments` to see the actual inputs required

Reply:
Should we just add a separate port arg? That seems to be what most apps use

 f.e iperf
 ```
 iperf -c 1.1.1.1 -p 80 
 ```

Reply:
Nick has a good point. It would simplify the argument parsing as well.

Reply:
Happy to. Given the repeating IP/Port arguments, I initially assumed that it'd be logical to consolidate; but it definitely does make sense to simplify. Here is what I can do:
`./go.sh SERVICE_ID -c CLIENT_IP -r CLIENT_PORT -s SERVER_IP -t SERVER_PORT -f SERVER_CONFIG [PRINT_DELAY]`

(-p already being used for print_delay)

Reply:
Adjusted

Reply:
Could we do something like `-cp` for client port and `-sp` for server port?

Reply:
To my knowledge, `getopt()` does not support "*long*" options, so I can only work with a single character. Currently looking into the difficulty of making a parser manually to support that

Reply:
It looks like making a command-line parser would not be in our best interest, meaning that we will have to limit ourselves to a single character per flag. I fear that making a parser from scratch would risk too many bugs from invalid inputs which are unaccounted for. I've checked `getopts()` and `argp()` as alternatives, but those seem to have the same issue. Also, `getopt()` does not directly support whitespace, but solving this issue may be easier than the latter. If you feel this is significantly better, I can look into it:
```
./go.sh SERVICE_ID -c CLIENT_IP CLIENT_PORT -s SERVER_IP SERVER_PORT -f SERVER_CONFIG [PRINT_DELAY]
```

Reply:
I forgot that getopt only supports single char arguments - my mistake. In that case, the original args you proposed are fine with me:
`./go.sh SERVICE_ID -c CLIENT_IP -r CLIENT_PORT -s SERVER_IP -t SERVER_PORT -f SERVER_CONFIG [PRINT_DELAY]`
Thanks!